src/api/streams/streamPostPolicy.js

```diff
@@ -0,0 +1,22 @@
+/* @flow strict-local */
+import type { StreamPostPolicy } from '../modelTypes';
+
+// See stream_post_policy at https://zulipchat.com/api/add-subscriptions
+export class StreamPostPolicies {
+  static everyone: 1 = 1;
+  static admins: 2 = 2;
+  static restrict_new_members: 3 = 3;
+
+  static getPolicy = (key: string): StreamPostPolicy => {
+    switch (Number(key)) {
+      case this.everyone:
+        return this.everyone;
+      case this.admins:
+        return this.admins;
+      case this.restrict_new_members:
+        return this.restrict_new_members;
+      default:
+        return this.everyone;
+    }
+  };
```

Comment:
OK, and now that I've read the whole thing -- I think this function can actually be dispensed with entirely! See my comment below about using a real `Map` vs. an object-as-Map. That will let us pass actual `StreamPostPolicy` values down through the layers here, instead of strings. And that should be able to go all the way to this line:
```
            <Touchable onPress={() => this.onOptionSelect(item.key)}>
```
so that `item.key` at this point is already of the right type and there's no need for the `onOptionSelect` callback to turn it back from a string.

We'll still need to stringify the keys to use as actual React keys, but that can be done just at this one line:
```
          keyExtractor={item => `${item.key}`}
```
and not interfere with the types anywhere else.

Reply:
```
keyExtractor={item => `${item.key}`}
```

^ flow won't allow this, throwing following error

```
Cannot coerce `item.key` to string because  `K` [1] should not be coerced.
```

But Flow is happy with `String(item.key)`, which is using `String` constructor
![image](https://user-images.githubusercontent.com/18511177/81735771-2d66f380-94b3-11ea-9a5a-0f52165ffc3d.png)